src/app/components/views/job-view/subviews/history-subview/history-subview.component.html

```diff
@@ -1,10 +1,10 @@
-<ng-container *ngIf="historyData">
-  <ng-container *ngFor="let history of historyData">
+<ng-container *ngIf="job && job.history">
+  <ng-container *ngFor="let history of job.history">
```

Comment:
I personally don't think it's needed but I twisted my arm and now graceland is happy--I added an elvis here.